src/testing/AddTestForm.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Box, Text, Label, Input, Checkbox, Button } from 'theme-ui';
+import { useFormik } from 'formik';
+
+import { CreateTestCommand, TestType, FilledSchema, ObjectSchema, FieldSchema } from '../api';
+
+const AnyBox = Box as any;
+
+export const AddTestForm = ({
+  testType,
+  onComplete,
+}: {
+  testType: TestType;
+  onComplete: (command: CreateTestCommand) => any;
+}) => {
+  // TODO: validation
+  // TODO: supporting more arbitrary schemas
```

Comment:
Main thing I would add is enums (selects/radios). Don't think we need to get into nested object, array, or oneOf/allOf.